### ISSUE TITLE
Allow non-ascii characters in jsonl

### ISSUE THREAD

**Issue by Muennighoff** [embeddings-benchmark/leaderboard#24]
In the jsonl files in the results chinese characters etc get turned into unicode codes but i think we should leave them as is; we're probably missing a `enforce_ascii=False` somewhere but cannot find where --- cc @orionw do you know where / do you agree on fixing this?

**Comment by orionw** [comment]
Yes, this would be in https://github.com/embeddings-benchmark/leaderboard/blob/main/refresh.py#L632 should be an easy fix.  

Is this an issue on the leaderboard space or just in the cached files? I can't seem to find any unicode chars in the space.

**Comment by Muennighoff** [comment]
Just for the cached files I think! Wdyt?

**Comment by orionw** [comment]
Ah great - makes sense to me! Just wanted to make sure I understood where the issue was :)
